test/diff-hl-test.el

```diff
@@ -126,6 +126,32 @@
     (should (looking-at "added"))
     (should-error (diff-hl-next-hunk) :type 'user-error)))
 
+(diff-hl-deftest diff-hl-indirect-buffer-move-with-thread ()
+  (diff-hl-test-in-source
+    (setq diff-hl-update-with-thread t)
```

Comment:
Update the name here too.

Also, please set it in `let`, so that it doesn't affect any other loaded tests.

Reply:
Thanks for the catch. Fixed.